src/components/Form/FormikSelectUser/FormikSelectUser.tsx

```diff
@@ -0,0 +1,69 @@
+import { useFormikContext } from 'formik'
+import Select from 'react-select'
+
+import { GebruikersInline } from '@/api/fetchers.schemas'
+
+export interface FormikSelectUserProps {
+    property:
+        | 'Eigenaar_1'
+        | 'Eigenaar_2'
+        | 'Portefeuillehouder_1'
+        | 'Portefeuillehouder_2'
+        | 'Opdrachtgever'
+    filter:
+        | 'Ambtelijk opdrachtgever'
+        | 'Behandelend Ambtenaar'
+        | 'Portefeuillehouder'
+    options: {
+        value: string
+        label: string
+        role: string
+    }[]
+}
+
+interface SelectUserValues {
+    Eigenaar_1: null | GebruikersInline
+    Eigenaar_2: null | GebruikersInline
+    Portefeuillehouder_1: null | GebruikersInline
+    Portefeuillehouder_2: null | GebruikersInline
+    Opdrachtgever: null | GebruikersInline
+}
+
+const FormikSelectUser = ({
```

Comment:
@Aidenbuis Why do we need a separate component for this? Can't you use the FormikSelect field from @pzh-ui/components?

Reply:
Will be tackled in the implement-gebiedsprogramma's branch